applications/virtual-fly-brain/client/src/shared/subHeader/SearchBuilder.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import useAutocomplete from '@mui/base/useAutocomplete';
+import { useAutocomplete } from '@mui/base/useAutocomplete';
```

Comment:
@Aiga115 this is giving error

Reply:
so without braces it was giving errors for me, is it giving you errors? @vidhya-metacell